examples/jenkins/pipeline/bluegreen-pipeline.yaml

```diff
@@ -59,7 +59,7 @@ objects:
             }
 
             stage("Go Live") {
-              sh "oc set -n ${project} route-backends ${appName} ${appName}-${tag}=100 ${appName}-${altTag}=0"
+              sh "oc set -n ${project} route-backends ${appName} ${appName}-${tag}=100 ${appName}-${altTag}=0 --loglevel=4"
```

Comment:
this is all jenkins job output, are we actually getting and displaying the jenkins job output somewhere in the test case?